--- libcdio-0.67/src/Makefile.am.orig	2004-02-28 13:45:02.000000000 +0100
+++ libcdio-0.67/src/Makefile.am	2004-03-05 00:15:53.819203328 +0100
@@ -44,14 +44,14 @@
 if BUILD_CDINFO_LINUX
 cdinfo_linux_SOURCES = cdinfo-linux.c
 cdinfo_linux_LDADD = $(LIBCDIO_LIBS) $(LIBPOPT_LIBS)
-bin_PROGRAMS = cd-info cd-read cdinfo-linux iso-info iso-read
+cdinfo_linux_prog = cdinfo-linux
 else
-bin_PROGRAMS = cd-info cd-read iso-info iso-read
 EXTRA_DIST = cdinfo-linux.c $(man_MANS)
 endif
+cdinfo_progs = cd-info cd-read iso-info iso-read
 else 
 EXTRA_DIST = cdinfo-linux.c cd-info.c cd-read.c $(man_MANS)
 man_MANS = 
 endif
-
+bin_PROGRAMS = $(cdinfo_progs) $(cdinfo_linux_prog)
 INCLUDES = -I$(top_srcdir) $(LIBPOPT_CFLAGS) $(LIBCDIO_CFLAGS) $(VCDINFO_CFLAGS)
