Description: Use ncursesw instead of ncurses

diff --git a/configure.ac b/configure.ac
index 5f9f9e78..7fcf6a64 100644
--- a/configure.ac
+++ b/configure.ac
@@ -668,8 +668,8 @@ fi
 if test "x$enable_cdda_player" = "xyes"; then
   PKG_CHECK_MODULES(CDDA_PLAYER, ncurses,
                     [AC_DEFINE(HAVE_KEYPAD, [1], [Define this if your libcurses has keypad])],
-                    [AC_CHECK_LIB(curses, keypad,
-                                  [CDDA_PLAYER_LIBS="-lcurses"],
+                    [AC_CHECK_LIB(ncursesw, keypad,
+                                  [CDDA_PLAYER_LIBS="-lncursesw"],
                                   [AC_MSG_WARN([Will not build cdda-player - did not find libncurses or libcurses])
                                     enable_cdda_player=no])])
 fi
